CHANGELOG.md

```diff
@@ -1025,6 +1319,38 @@
   [#11431](https://github.com/mixxxdj/mixxx/pull/11431)
   [#11434](https://github.com/mixxxdj/mixxx/pull/11434)
   [#11301](https://github.com/mixxxdj/mixxx/pull/11301)
+  [#11688](https://github.com/mixxxdj/mixxx/pull/11688)
+  [#11685](https://github.com/mixxxdj/mixxx/pull/11685)
+  [#12000](https://github.com/mixxxdj/mixxx/pull/12000)
+  [#12003](https://github.com/mixxxdj/mixxx/pull/12003)
+  [#11684](https://github.com/mixxxdj/mixxx/pull/11684)
+  [#12031](https://github.com/mixxxdj/mixxx/pull/12031)
+  [#12008](https://github.com/mixxxdj/mixxx/pull/12008)
+  [#12054](https://github.com/mixxxdj/mixxx/pull/12054)
+  [#11768](https://github.com/mixxxdj/mixxx/pull/11768)
+  [#11893](https://github.com/mixxxdj/mixxx/pull/11893)
+  [#11636](https://github.com/mixxxdj/mixxx/pull/11636)
+  [#12026](https://github.com/mixxxdj/mixxx/pull/12026)
+  [#12230](https://github.com/mixxxdj/mixxx/pull/12230)
+  [#12375](https://github.com/mixxxdj/mixxx/pull/12375)
+  [#11619](https://github.com/mixxxdj/mixxx/pull/11619)
+  [#11917](https://github.com/mixxxdj/mixxx/pull/11917)
+  [#11936](https://github.com/mixxxdj/mixxx/pull/11936)
+  [#12033](https://github.com/mixxxdj/mixxx/pull/12033)
+  [#12263](https://github.com/mixxxdj/mixxx/pull/12263)
+  [#12212](https://github.com/mixxxdj/mixxx/pull/12212)
+  [#12066](https://github.com/mixxxdj/mixxx/pull/12066) [#12065](https://github.com/mixxxdj/mixxx/issues/12065)
+
+### Known issues
+
+* Volume / Loudness spikes on Windows with M4A/AAC files.
+  Last known working version is Windows 10 build 17763.
+  Affected versions are Windows 10 build 19041 and Windows 11 build 22000.
+  A fix is promised in 2024-02 with Windows 11 build 26016.
+  [#12289](https://github.com/mixxxdj/mixxx/issues/12289)
+  [#11094](https://github.com/mixxxdj/mixxx/issues/11094)
+* Different, language depending library sorting on macOS
+  [#12517](https://github.com/mixxxdj/mixxx/issues/12517)
```

Comment:
I remember we came across an issue recently that we wanted to add to 'known issues' (but we didn't think of this place). I added the Changelog label and added it to the 2.4 milestone, but I can't find it anymore...

Maybe someone else remembers what that was.
I think @fwcd and/or @m0dB were involved.